widget/ui/src/components/TokenAmount/TokenAmount.tsx

```diff
@@ -24,24 +25,30 @@ export function TokenAmount(props: PropTypes) {
         />
         <div>
           {props.label && (
-            <Typography ml={4} size="xsmall" variant="body" color="$neutral700">
-              {props.label}
-            </Typography>
+            <div style={{ display: 'flex' }}>
```

Comment:
This looks a working around to me. I think `Divider` should works here correctly (if not, we should fix that component) or it should be fixed somehow in layout layer of this component.

Reply:
There are some other places that has this working around, Same comment applies to them as well. I don't put comments on them to keep comments cleaner.  Please make sure you're checking them as well.

Reply:
I wasn't really sure what this meant. Are you saying that the divider itself should to support the display flex?